src/Datadog.Trace/Configuration/IntegrationIds.cs

```diff
@@ -13,6 +13,7 @@ internal enum IntegrationIds
         AspNetMvc,
         AspNetWebApi2,
         GraphQL,
+        Msmq,
```

Comment:
I don't think it would cause any version conflict, but I'd feel safer if you declared it last, so that the numerical id of the existing integrations don't change